check-osp/utils.py

```diff
@@ -28,3 +28,12 @@ def ssh_client(hostname, username, password):
         # authenticates, and then creates stream tunnels, called channels, across the session.
         # If connection is not established, returns None.
         raise timeout("No pingable IP found")
+
+
+def get_services_status_list(ssh):
+    """
+    This function fetches list of all units of type services from given host and returns in dict.
+    """
+    stdin, stdout, stderr = ssh.exec_command("systemctl list-units --type service --state=running"
+    " --state=exited --no-page   --no-legend | awk -F ' ' '{print $1 \": \" $3 \" (\" $4\")\"}'")
```

Comment:
It would be nice to add some comment how it works here :)

Reply:
Yep. Adding.